snippets/wasm/getting_started.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  defaultConfig,
+  connect,
+  type SdkEvent,
+  setLogger,
+  type BindingLiquidSdk,
+  type LogEntry
+} from '@breeztech/breez-sdk-liquid'
+
+// Init stub
+const init = async () => {}
+
+const exampleGettingStarted = async () => {
+  // ANCHOR: init-sdk
+  const mnemonic = '<mnemonics words>'
+
+  // Call init when using the SDK in a web environment before calling any other SDK
+  // methods. This is not needed when using the SDK in a Node.js/Deno environment.
+  //
+  // import init, { defaultConfig, connect } from '@breeztech/breez-sdk-liquid'
+  await init()
+
+  // Create the default config, providing your Breez API key
+  const config = defaultConfig('mainnet', '<your-Breez-API-key>')
+
+  // The `workingDir` does not need to be set in a web environment
```

Comment:
On the browser the working dir is still used to build the names of the indexedDb databases. If someone want to run concurrent instances using different wallets they should provide distinct working dirs

Reply:
Forget that, that's even already the case but I had forgotten.